packages/elements/src/tree/__demo__/index.html

```diff
@@ -147,6 +147,29 @@
       <button uncheck>Uncheck All</button>
     </demo-block>
 
+    <demo-block header="Query" layout="normal" tags="query">
+      <input id="inputQuery" type="text" placeholder="Input query" />
+      <ef-tree id="queryTree" ></ef-tree>
+      <button expand>Expand All</button>
+      <button collapse>Collapse All</button>
+      <button check>Check All</button>
+      <button uncheck>Uncheck All</button>
+      <script>
+        const debounce = (callback, timeout = 500) => {
+          let timer = setTimeout(callback, timeout);
+          return (...args) => {
+            clearTimeout(timer);
+            timer = setTimeout(() => callback.apply(this, args), timeout);
+          };
+        };
+        const el = document.getElementById('queryTree');
+        const setQuery = debounce((e) => {
+          el.query = e.target.value;
```

Comment:
This line is failing in the demo page on page load

Reply:
Fixed

Reply:
Fixed